WIP. running all tests for various ffmpeg versions (#9552)
cvat-ai/cvat: utils/switch_task_to_static/README.md

Reviewer: I suggest to rename the directory to a more problem-focused name, like "ffmpeg_compatibility" or "ffmpeg_upgrade_helpers".

# Description
In version 2.46.0, CVAT upgraded the FFMPEG library it uses to split videos into frames from 4.3.1 to 8.0.

> There is a small chance that some video files may be splitted to frames differently by different FFMPEG versions.

Reviewer 2: ```suggestion
There is a small chance that some video files may be split into frames differently by different FFmpeg versions.
```

> In case of any difference in frame decoding,
> this script may be used to switch a task to static chunks and generate frames with the old FFMPEG version.

Reviewer: ```suggestion
In the case of any difference in frame decoding,
this script may be used to switch a task to static chunks and generate frames with the old FFMPEG version. This option requires administrator
access to the server instance. If you do not have such access, please try 
to contact the server administration.
```

# Usage

If your CVAT is deployed through docker, run
```shell

> docker compose -f ./utils/switch_task_to_static/docker-compose.yaml run --rm generate_chunks_for_task <task id>

Reviewer: ```suggestion
docker compose \
  -f docker-compose.yml \
  \  # optionally -f docker-compose.dev.yml \
  -f ./utils/switch_task_to_static/docker-compose.yaml \
  run --rm generate_chunks_for_task <task_id>
```

Doing so will allow to reuse network and volume definitions
